workshops/templates/workshops/admin_dashboard.html

```diff
@@ -39,6 +40,8 @@
     <table class="table table-striped">
     {% for event in current_events %}
     <tr>
+      <td class="text-center" width="25px">{% if event.is_swc %}{% bootstrap_tag "SWC" %}{% endif %}</td>
+      <td class="text-center" width="25px">{% if event.is_dc %}{% bootstrap_tag "DC" %}{% endif %}</td>
```

Comment:
@chrismedrela is there a quick fix using a loop over available types of carpentry? (I have no comment on the amount of space required.)